server/src/main/scala/controllers/Application.scala

```diff
@@ -560,7 +560,7 @@ class Application @Inject()(implicit val config: Configuration,
       f"${pit.getHours()}%02d-${pit.getMinutes()}%02d-hours-$startHour%02d-to-$endHour%02d"
 
 
-    exportDesksToCSV(terminalName, pit, startHour, endHour).map {
+    exportDesksToCSV(terminalName, pit, startHour, endHour, loadBestCrunchStateForPointInTime(pit.millisSinceEpoch)).map {
```

Comment:
i'd be tempted to move the call to loadBestCrunchStateForPointInTime() to a val with a shorter name to keep this call a bit shorter